t/mojo/morbo.t

```diff
@@ -136,6 +136,49 @@ subtest 'Broken symlink' => sub {
   ok !$warned, 'no warnings';
 };
 
+subtest 'Randomized Request IDs' => sub {
+  my @ids;
+  my $src = <<'EOF';
+use Mojolicious::Lite;
+
+app->log->level('fatal');
+
+# Return "request_id"
+get '/request_id' => sub {
+  my $c = shift;
+  $c->render( text => $c->req->request_id );
+};
+
+app->start;
+EOF
+
+  # Fetch initial request_ids
+  $script->spurt($src);
+  sleep 3;
+
+  $tx = $ua->get("http://127.0.0.1:$port/request_id");
+  like $tx->res->body, qr/^[\-_0-9a-z]+$/i, 'okay content';
```

Comment:
That's an inconsistent generic test description.

Reply:
*"Fixed in the Next Release"*

(With `request_id is valid` and 'request_ids differ', respectively)